### PR TITLE
Create scheduled scans for nessus

### PR DESCRIPTION
**Pull request by akinnane**
Relates to #12

### CODE REVIEW

File: schedule_scans.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            "text_targets": scan["text_targets"],
+            "agent_group_id": [],
+        },
+    }
+
+
+def create_gds_scans(config):
+    for scan in config.values():
+        s = create_scan_config(scan)
+        p(s)
```
Comment (alice-carr) [inline schedule_scans.py:108]: can we get rid of this print?

File: schedule_scans.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    with open("scan_config/scan.toml", "r") as f:
+        return toml.load(f)
+
+
+def main():
+    # set_policy(base_url, custom_headers)
+    # p(policy_details(base_url, custom_headers, 7))
+    # p(create_policy(base_url, custom_headers))
+    # p(list_policy_templates())
+    # p(advanced_dynamic_policy_template_uuid())
```
Comment (alice-carr) [inline schedule_scans.py:127]: can we get rid of these comments?

File: schedule_scans.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+    config = load_scan_config()
+    create_gds_scans(config)
+
+
+if __name__ == "__main__":
+    main()
```
Comment (alice-carr) [inline schedule_scans.py:134]: we're not testing this file in this PR. Are we planning to later?

File: schedule_scans.py (hunk 1 above)
Comment (alice-carr) [reply schedule_scans.py:108]: (and also remove the import pprint at the top if we do this)

File: schedule_scans.py (hunk 3 above)
Comment (pritchyspritch) [reply schedule_scans.py:85]: Will add later I think

File: process_scans.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                {
+                    "timestamp": scan["last_modification_date"] * 1000,
+                    "message": ",".join(row).replace("\n", " "),
+                }
+            )
+
+            # Send in batches of 10_000
+            if len(events) >= 9999:
+                token = logs_client().put_log_events(
+                    logGroupName="/gds/nessus-data",
```
Comment (denizgenc) [inline process_scans.py:50]: Don't see why we can't re-use `group_name` here (like we do with `stream_name` on the line below):
```suggestion
                    logGroupName=group_name,
```

**Pull request merged**